src/backend/blockchain/src/routes.py

```diff
@@ -25,30 +25,39 @@ def get_chain():
     return jsonify(length=len(chain), chain=chain), HttpCode.OK.value
 
 
-@app.route('/proof', methods=['POST'])
+@app.route('/addBlock', methods=['POST'])
 def proof():
     data = request.get_json()
-    proof = '0' * Blockchain.difficulty
-    # proof = data["proof"]
     try :
         new_transaction = Transaction(
-            data["from"], 
-            data["to"], 
+            data["from"],
+            data["to"],
             data["amount"]
         )
+        miner_id = data["minerId"]
+        _hash = data["proof"]
```

Comment:
Should this be _hash or hash?

Reply:
hash seems to a python protected variable so changed it to _hash instead

Reply:
clarification: hash() is an inbuilt method in python so cannot use that name.